### ISSUE TITLE
ValueError: need more than 0 values to unpack in execsnoop

### ISSUE THREAD

**Issue by bobrik** [iovisor/bcc#473]
I've changed the code to print the line causing the issue:

```python
print(line + " | " + line[:ts_end])
pid, cpu, flags, ts = line[:ts_end].split()
```

It happens when I install new packages on ubuntu 16.04 running in vagrant.

```
4677  [001] d...  1118.654183: : ARG /usr/bin/install-info | 4677  [001] d...  1118.654183
4677  [001] d...  1118.654195: : ARG /usr/share/info/nano.info.gz | 4677  [001] d...  1118.654195
4677  [001] d...  1118.654196: : ARG /usr/share/info/dir | 4677  [001] d...  1118.654196
4677  [000] d...  1118.655977: : RET 0 | 4677  [000] d...  1118.655977
update-info-dir  4677     0 /usr/bin/install-info /usr/share/info/nano.info.gz /usr/share/info/dir
4678  [001] d...  1118.657008: : ARG | 4678  [001] d...  1118.657008
4678  [001] d...  1118.657024: : ARG | 4678  [001] d...  1118.657024
4678  [001] d...  1118.657024: : ARG gzip -d | 4678  [001] d...  1118.657024
4678  [000] d...  1118.658441: : RET 0 | 4678  [000] d...  1118.658441
install-info     4678     0 gzip -d
4679  [000] d...  1118.659031: : ARG /bin/gzip | 4679  [000] d...  1118.659031
4679  [000] d...  1118.659064: : ARG -d | 4679  [000] d...  1118.659064
4679  [000] d...  1118.659206: : RET 0 | 4679  [000] d...  1118.659206
<...>            4679     0 /bin/gzip -d
4680  [001] d...  1118.663486: : ARG /var/lib/dpkg/info/man-db.postinst | 4680  [001] d...  1118.663486
4680  [001] d...  1118.663516: : ARG triggered | 4680  [001] d...  1118.663516
4680  [001] d...  1118.663573: : ARG /usr/share/man | 4680  [001] d...  1118.663573
4680  [000] d...  1118.664995: : RET 0 | 4680  [000] d...  1118.664995
dpkg             4680     0 /var/lib/dpkg/info/man-db.postinst triggered /usr/share/man
4681  [000] d...  1118.665806: : ARG /usr/bin/perl | 4681  [000] d...  1118.665806
4681  [000] d...  1118.665831: : ARG -e | 4681  [000] d...  1118.665831
4681  [000] d...  1118.665833: : ARG @pwd = getpwnam("man"); $) = $( = $pwd[3]; $> = $< = $pwd[2]; | 4681  [000] d...  1118.665833
 |
Traceback (most recent call last):
  File "/usr/share/bcc/tools/execsnoop", line 127, in <module>
    (task, pid, cpu, flags, ts, msg) = b.trace_fields()
  File "/usr/lib/python2.7/dist-packages/bcc/__init__.py", line 584, in trace_fields
    pid, cpu, flags, ts = line[:ts_end].split()
ValueError: need more than 0 values to unpack
```

**Comment by brendangregg** [comment]
This would be due to execsnoop using the old trace_pipe approach, and not handling unexpected input correctly (other things on trace_pipe). That should be fixed, then execsnoop should be moved to tools/old, with a new version of execsnoop written that uses the perf_output mechanism in 4.5.

**Comment by markdrayton** [comment]
This should be fixed by #555.

**Comment by brendangregg** [comment]
@markdrayton thanks, yes, closing.
